clumio/resource_aws_connection.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2021. Clumio, Inc.
+
+// clumio_aws_connection definition and CRUD implementation.
+package clumio
+
+import (
+	"context"
+	"strings"
+
+	aws_connections "github.com/clumio-code/clumio-go-sdk/controllers/aws_connections"
+	"github.com/clumio-code/clumio-go-sdk/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// clumioAWSConnection returns the resource for Clumio AWS Connection.
+
+func clumioAWSConnection() *schema.Resource {
+	return &schema.Resource{
+		// This description is used by the documentation generator and the language server.
```

Comment:
Oh I just noticed this ... can this document be generated?

Reply:
No. This will be used to generate the documentation for the provider.

Reply:
Right, I was more seeing if we could go ahead and generate this documentation which is seems like you have now.